utils/scripts/ocm/ocm.py

```diff
@@ -1145,12 +1178,13 @@ def compare_with_old_version_file(self):
         "releases/download/v0.1.55/ocm-linux-amd64"
     )
     parser = argparse.ArgumentParser(
+        usage=argparse.SUPPRESS,
```

Comment:
what does it change?

Reply:
argparse.SUPPRESS is used to handle hidden arguments. This is required for showing the expected help content since we used if else stmt in between user inputs.